Modules/Migration.lua

```diff
@@ -138,9 +138,6 @@ local migrationFunctions = {
         Questie.db[Questie.db.global.questieTLoc].TrackerLocation = nil
     end,
     [14] = function()
```

Comment:
Lets just remove this step entirely 😁 

```suggestion
```

Reply:
Oh right... I left that in there to avoid any errors. :D I can update that and increment the table keys. It'll make it into my next check-in on my PR.

Reply:
Not sure if we should increment the table keys, cause that might cause settings changes for users. Just removing it should be fine 👍🏻

Reply:
I removed #14 and renumbered them. I looked over that stack and it's not going to break anything. Matter of fact, some of the Questie.db keys have changed so I think I should add the new ones in? I honestly don't think it's needed since there isn't anything saved to the old keys that needs to be migrated.

It didn't break anything for me when I changed the DB keys and reloaded my UI. It just simply worked.